tests/integration/tests/test_sanity_checks.py

```diff
@@ -88,7 +89,28 @@ def test_cspm_findings(elastic_client, match_type):
         AssertionError: If the resource type is missing.
     """
     query_list = [{"term": {"rule.benchmark.id": "cis_aws"}}, {"term": {"resource.type": match_type}}]
-    query, sort = elastic_client.build_es_must_match_query(must_query_list=query_list, time_range="now-24h")
+    query, sort = cspm_client.build_es_must_match_query(must_query_list=query_list, time_range="now-24h")
 
-    results = get_findings(elastic_client, CONFIG_TIMEOUT, query, sort, match_type)
+    results = get_findings(cspm_client, CONFIG_TIMEOUT, query, sort, match_type)
+    assert len(results) > 0, f"The resource type '{match_type}' is missing"
+
+
+@pytest.mark.sanity
+def test_cnvm_findings(cnvm_client):
```

Comment:
Just for consistency and clarity, I would keep it in the same format (which means we need to update `test_data` on line 15) but your call
```suggestion
@pytest.mark.sanity
@pytest.mark.parametrize("match_type", tests_data["cnvm"])
def test_cnvm_findings(cnvm_client):
```